src/snowflake/connector/auth/_auth.py

```diff
@@ -663,7 +668,7 @@ def unlock_temporary_credential_file() -> bool:
         return False
 
 
-def delete_temporary_credential(host, user, cred_type):
+def delete_temporary_credential(host, user, cred_type) -> None:
```

Comment:
I'm confused, why this has `-> None` but in `read_temporary_credential_file` the `-> None` is removed?

Reply:
That function actually returns something while this one doesn't. Instead of fixing the return type of `read_temporary_credential_file` I just removed `-> None` for now.

Reply:
I see. Thanks for the clarification